.github/workflows/linting.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Linting"
+
+on:
+  push:
+    paths:
+      - './**'
+      - '.github/workflows/**'
+  pull_request:
+    branches: [ dev, main ]
+    paths:
+      - './**'
+
+jobs:
+  # Non-blocking job that checks .md file locations and also lints them
+  markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v2
+      - name: "Validate markdown file locations"
+        continue-on-error: true
+        run: "scripts/check-markdown-files.sh" 
```

Comment:
qq: Is the intent of running the script to flag where the `md` files should be located? If that is the case, it would be good to mention that in the script.

Reply:
This script would be blocking by itself but then the action chooses to continue on error. So I'll make that note in the action